htdocs/core/lib/pdf.lib.php

```diff
@@ -724,11 +724,13 @@ function pdf_pagehead(&$pdf, $outputlangs, $page_height)
 		$filepath = $conf->mycompany->dir_output.'/logos/'.$conf->global->MAIN_USE_BACKGROUND_ON_PDF;
 		if (file_exists($filepath)) {
 			$pdf->SetAutoPageBreak(0, 0); // Disable auto pagebreak before adding image
+			if (getDolGlobalString('MAIN_USE_BACKGROUND_ON_PDF_ALPHA')) { $pdf->SetAlpha($conf->global->MAIN_USE_BACKGROUND_ON_PDF_ALPHA); } // Option for change opacity of background
 			$pdf->Image($filepath, (isset($conf->global->MAIN_USE_BACKGROUND_ON_PDF_X) ? $conf->global->MAIN_USE_BACKGROUND_ON_PDF_X : 0), (isset($conf->global->MAIN_USE_BACKGROUND_ON_PDF_Y) ? $conf->global->MAIN_USE_BACKGROUND_ON_PDF_Y : 0), 0, $page_height);
-			$pdf->SetPageMark(); // This option avoid to have the images missing on some pages
+			if (getDolGlobalString('MAIN_USE_BACKGROUND_ON_PDF_ALPHA')) { $pdf->SetAlpha(1); }
 			$pdf->SetAutoPageBreak(1, 0); // Restore pagebreak
 		}
 	}
+	$pdf->SetPageMark(); // This option avoid to have the images missing on some pages
```

Comment:
Why moving this outside of the if (!empty($conf->global->MAIN_USE_BACKGROUND_ON_PDF)
If this test is wrong nothing is modified, so do we really need to do a SetPageMark ?

Reply:
Hello @eldy,
I deleted my change.

Yes, we need $pdf->SetPageMark(); otherwise the image is not really considered as a background image : the layers are mixed between the background image, the text, the frames, etc

Best regards,
Alban

Reply:
Revision 2 : I confirm that when we use a PDF as a background image, we have the same problem as with MAIN_USE_BACKGROUND_ON_PDF